benchmark/bootstrap/aws-launch.sh

```diff
@@ -58,7 +58,7 @@ show_usage() {
     echo "  --firewood-branch BRANCH    Firewood git branch to checkout"
     echo "  --avalanchego-branch BRANCH AvalancheGo git branch to checkout"
     echo "  --coreth-branch BRANCH      Coreth git branch to checkout"
-    echo "  --libevm-branch BRANCH      LibEVM git branch to checkout"
+    echo "  --libevm-commit COMMIT      LibEVM git commit to checkout"
```

Comment:
Technically these are all commits not branches. Why are we renaming this one without renaming the others?